src/Routing/Loader.php

```diff
@@ -40,6 +45,10 @@ public function load($resource, $type = null) : RouteCollection
         return $routes->build();
     }
 
+    /**
+     * @param mixed $resource
+     * @param ?string $type
```

Comment:
`?string` cannot be put directly into the signature because it's inherited I guess

Reply:
Yep. (Should be `string|null` here too.)

Reply:
Though the parent interface defines these so shouldn't be needed?

Reply:
Reading https://github.com/phpstan/phpstan-strict-rules/issues/51#issuecomment-441710858 `{@inheritdoc}` is required, but isn't in the next version (just tried and confirmed).

Reply:
`{@inheritdoc}` works for `ResourceComparator`/`TraversableComparator` but not `Loader`? 🤔